extensions/algebra/circuit/src/lib.rs

```diff
@@ -40,7 +40,11 @@ pub struct FieldExprVecHeapStep<
     const NUM_READS: usize,
     const BLOCKS: usize,
     const BLOCK_SIZE: usize,
->(FieldExpressionStep<Rv32VecHeapAdapterStep<NUM_READS, BLOCKS, BLOCKS, BLOCK_SIZE, BLOCK_SIZE>>);
+>(
+    pub  FieldExpressionStep<
```

Comment:
do you need `pub`? I put `as_ref()` and `as_mut()` below if that's enough

Reply:
actually no. i missed the deref